test/active_record/active_record_test_helper.rb

```diff
@@ -0,0 +1,9 @@
+require "test_helper"
```

Comment:
Could you help me add the magic comment `# frozen_string_literal: true` to the first line of this file?